src/components/ui/piechart.tsx

```diff
@@ -0,0 +1,81 @@
+const Progress = ({ progressPercent }: { progressPercent: number }) => {
+  // size of the circle
+  const size = 128;
+  // radius of the circle
+  const radius = 40;
+  // circumference of the circle
+  const circumference = 2 * Math.PI * radius;
+  // strokeDashoffset
+  const strokeDashoffset =
+    circumference - (progressPercent / 100) * circumference;
+
+  return (
+    <>
+      <style>
+        {`@keyframes circleStroke {
+          from {
+            stroke-dashoffset: ${circumference};
+          }
+          to {
+            stroke-dashoffset: ${strokeDashoffset};
+          }
+        }
+        @keyframes rotationObject {
+          from {
+            transform: rotate(0deg);
+          }
+          to {
+            transform: rotate(${progressPercent * 3.6}deg)
+          }
+        }
+        `}
+      </style>
+      <div className="relative">
+        <svg
+          viewBox={`0 0 ${size} ${size}`}
+          style={{ transform: 'rotate(-90deg)' }}
```

Comment:
Could you modify this code with TailwindCSS class?
`className="transform rotate-90"`